src/scripting_api/aform.js

```diff
@@ -37,6 +37,11 @@ class AForm {
       "m/d/yy HH:MM",
     ];
     this._timeFormats = ["HH:MM", "h:MM tt", "HH:MM:ss", "h:MM:ss tt"];
+
+    // https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
+    // The following regex has been copy/paste from the above link.
```

Comment:
I think the following would be a bit easier to read, with the URL below the description:

```javascript
# The e-mail address regex below originates from:
# https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
```